app/cli/cmd/workflow_workflow_run_list.go

```diff
@@ -105,7 +105,7 @@ func workflowRunListTableOutput(runs []*action.WorkflowRunItem) error {
 
 	for _, p := range runs {
 		wf := p.Workflow
-		r := table.Row{p.ID, wf.NamespacedName(), versionString(p.ProjectVersion), p.State, p.CreatedAt.Format(time.RFC822), p.RunnerType}
+		r := table.Row{p.ID, wf.NamespacedName(), p.ProjectVersion.Version, p.ProjectVersion.Prerelease, p.State, p.CreatedAt.Format(time.RFC822), p.RunnerType}
```

Comment:
thanks for finding the bug, but why did you decide to fill up the pre-release column instead of removing the column header?

note that `versionString` already returned pre-release status in a way it's consistent during attestation process for example.

I do not feel that using an extra column is worth the real-state, and probably that's why I left it behind but forgot to remove the header.

Reply:
Just because I saw the column without values and checking the logs I saw it was added but never filled. In any case, I've created a PR to remove it and put the old behavior back: https://github.com/chainloop-dev/chainloop/pull/1881

Reply:
thanks @javirln